examples/notebooks/tour-of-beam/windowing.ipynb

```diff
@@ -0,0 +1,703 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7WkYLzFCo4Rl"
+      },
+      "source": [
+        "# Fixed time windows\n",
+        "\n",
+        "If we want to analyze our data hourly, daily, monthly, etc. We might want to create evenly spaced intervals.\n",
+        "\n",
+        "[`FixedWindows`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.window.html#apache_beam.transforms.window.FixedWindows)\n",
+        "allow us to create fixed-sized windows.\n",
+        "We only need to specify the _window size_ in seconds.\n",
+        "\n",
+        "In Python, we can use [`timedelta`](https://docs.python.org/3/library/datetime.html#timedelta-objects)\n",
+        "to help us do the conversion of minutes, hours, or days for us.\n",
+        "\n",
+        "> ℹ️ Some time deltas like a month cannot be so easily converted into seconds, since a month can have from 28 to 31 days.\n",
```

Comment:
I think you can either:
- Remove the month concept, drop this comment and and all the other approximately x months comments, and simply say 30 day, 90 day etc.
- Or introduce custom windows later on which can do more sophisticated things.

Probably the first one (simplifying) would be better.

Reply:
Makes sense, I'm changing all the months mentions to days.